tests/unit/test_cesm_driver.py

```diff
@@ -1,4 +1,5 @@
 import os
+from sys import platform
```

Comment:
Do we also need to add `global_param` examples for the CESM driver as we did for classic and image drivers? Or did I miss something?